src/helpers/create-jwt-token.js

```diff
@@ -0,0 +1,11 @@
+const jwt = require("jsonwebtoken");
+const config = require("../../config");
+
+module.exports = (options = {}) => {
+  return jwt.sign("", config.jwtPrivateKey, options);
```

Comment:
do not use empty string 
you should put user id to jwt

Reply:
this function generates only a token it does not need a user ID